docs/admin/troubleshooting.md

```diff
@@ -414,6 +414,271 @@ Verify if you're using Infiniband for `fabric_iface`: in the server config. The
+        No errors found on PMem module 0x0021
+        No errors found on PMem module 0x0101
+        No errors found on PMem module 0x0111
+        No errors found on PMem module 0x0121
+        No errors found on PMem module 0x1001
+        No errors found on PMem module 0x1011
+        No errors found on PMem module 0x1021
+        No errors found on PMem module 0x1101
+        No errors found on PMem module 0x1111
+        No errors found on PMem module 0x1121
+
+        # ipmctl show -error Media
+         DimmID | System Timestamp    | Error Type
+        =============================================================
+         0x0001 | 01/12/2022 21:18:53 | 0x04 - Locked/Illegal Access
+         0x0001 | 01/12/2022 21:18:53 | 0x04 - Locked/Illegal Access
+         ....
+         ....
+         ....
+         0x1121 | 02/03/2022 16:50:17 | 0x04 - Locked/Illegal Access
+         0x1121 | 02/03/2022 16:50:17 | 0x04 - Locked/Illegal Access
+        Show Error executed successfully
+
+### ndctl
+
+NDCTL is another utility library for managing the PCMEM. It can detect the media errors and scrub it to avoide accesses that could lead to uncorrectable memory error handling events.
+[NDCTL user guide](https://docs.pmem.io/ndctl-user-guide/) has more details about the utility.
+
+#### Read bad blocks data from sys on clean system.
+
+	# cat /sys/block/pmem*/badblocks
+	#
+
+#### [ndctl list](https://docs.pmem.io/ndctl-user-guide/ndctl-man-pages/ndctl-list) command on clean system.
+
+        # ndctl list -M
+        [
+          {
+                "dev":"namespace1.0",
+                "mode":"fsdax",
+                "map":"dev",
+                "size":3183575302144,
+                "uuid":"c0d02d75-2629-4393-ae91-44e914c82e7d",
+                "sector_size":512,
+                "align":2097152,
+                "blockdev":"pmem1",
+          },
+          {
+                "dev":"namespace0.0",
+                "mode":"fsdax",
+                "map":"dev",
+                "size":3183575302144,
+                "uuid":"7294d1c0-2145-436b-a2af-3115db839832",
+                "sector_size":512,
+                "align":2097152,
+                "blockdev":"pmem0"
+          }
+        ]
+
+#### [ndctl start-scrub](https://docs.pmem.io/ndctl-user-guide/ndctl-man-pages/ndctl-start-scrub) command ran on system which has bad blocks.
+
+	# ndctl start-scrub
+	[
+	  {
+		"provider":"ACPI.NFIT",
+		"dev":"ndbus0",
+		"scrub_state":"active",
+		"firmware":{
+		  "activate_method":"reset"
+		}
+	  }
+	]
+
+#### [ndctl wait-scrub](https://docs.pmem.io/ndctl-user-guide/ndctl-man-pages/untitled-2) command ran on system which has bad blocks. Scrubbing takes time and this command will take longer time (~8-10 min) based on number of PCMEM on the system.
+
+	# ndctl wait-scrub
+	[
+	  {
+		"provider":"ACPI.NFIT",
+		"dev":"ndbus0",
+		"scrub_state":"idle",
+		"firmware":{
+		  "activate_method":"reset"
+		}
+	  }
+	]
```

Comment:
Maybe probably more details on what it does and when it would be needed in a normal DAOS installation.

Reply:
Added few line but we may need to add more test scenarios on how this scrubbing will work and how application will react before and after scrub.